analyzers/its/create-issue-reports.ps1

```diff
@@ -108,10 +108,12 @@ function GetActualIssues([string]$sarifReportPath){
         return
     }
 
-    # Change spaces to %20
+    # Change spaces to %20 and replace temporary paths
+    $tempFileNameRegex = '^.*\.(NETFramework|NETCoreApp),Version='
     $allIssues.location | Foreach-Object {
         if ($_.uri) {
             $_.uri = $_.uri.replace(' ', '%20')
+            $_.uri = [System.Text.RegularExpressions.Regex]::Replace($_.uri, $tempFileNameRegex, 'Replaced-Temporary-Path\.$1,Version=')
```

Comment:
I had to remove this hack from `regression-test.ps1`, because it first sorted the issues by URI (so the temp path was sometimes before normal path and sometimes it wasn't) and then did the replace. Actual temp filename information was lost on the way.

New version does the replacement before sorting to make it deterministic. And it also preserves the actual temp file name (replaces only the temp directory) so we can see what file is it from (like `Replaced-Temporary-Path\.NETCoreApp,Version=v3.1.AssemblyAttributes.cs`).